scripts/ci/check_compliance.py

```diff
@@ -561,6 +561,36 @@ def parse_kconfig(self, filename="Kconfig", hwm=None):
             # Clean up the temporary directory
             shutil.rmtree(kconfiglib_dir)
 
+    def get_logging_syms(self, kconf):
+        # Returns a set() with the names of the Kconfig symbols generated with
+        # logging template in samples/tests folders. The Kconfig symbols doesn't
+        # include `CONFIG_` and for each module declared 4 Kconfig symbols are
+        # created:
+        #   - *_LOG_LEVEL
+        #   - *_LOG_LEVEL_DBG
+        #   - *_LOG_LEVEL_ERR
+        #   - *_LOG_LEVEL_INF
+        #   - *_LOG_LEVEL_WRN
+
+        # Warning: Needs to work with both --perl-regexp and the 're' module.
+        regex = r"^\s*(?:module\s*=\s*)([A-Z0-9_]+)\s*(?:#|$)"
```

Comment:
This will match all variable definitions named 'module', even if they are not used with a `source "subsys/logging/Kconfig.template.log_config"`. It will also match in help text, etc.. If we don't care, we should add a comment saying that.

Reply:
It's done like that in `get_defined_syms`. IMO it's fine, I didn't find something else that uses that `module` to define a Kconfig symbol via a template. And I think the chance of getting that pattern in a help text is quite low